source/appModules/soffice.py

```diff
@@ -151,7 +92,9 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 
 		# optimisation: Assume a hyperlink occupies a full attribute run.
 		try:
-			if obj.IAccessibleTextObject.QueryInterface(IA2.IAccessibleHypertext).hyperlinkIndex(offset) != -1:
+			if obj.IAccessibleTextObject.QueryInterface(
+				IAccessibleHandler.IA2.IAccessibleHypertext

```

Comment:
Any particular reason why you prefer `IAccessibleHandler.IA2.*` over just `IA2.*`?

Reply:
Not really, other than that I no longer want to import the typelib stuff from the comInterfaces directory.